### ISSUE TITLE
Weird sorting in `Bower: Install`

### ISSUE THREAD

**Issue by sindresorhus** [benschwarz/sublime-bower#25]
`jquery` should be at the top:

![Screen Shot 2013-04-07 at 00 44 58](https://f.cloud.github.com/assets/170270/347937/b73b7388-9f0b-11e2-8e5f-1390d5294fe8.png)

**Comment by benschwarz** [comment]
Yeah… I'm doing a reverse order of whatever the API returns from the list method.
So unless we add an issue to make the API's `/packages` endpoint list them in some sensical order, then I'll need to do some manipulation.

**Comment by sindresorhus** [comment]
I wouldn't wait for it, the server is going to be rewritten in Node, but it will probably take some time.
